src/components/BarSelectionProperties/BarSelectionProperties.scss

```diff
@@ -19,5 +19,5 @@
     padding-bottom: 10px;
 }
 .phones-properties-list{
-    padding: 10px 0;
+    padding: 7px 0;
 }
```

Comment:
https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files